buda/entities/payment_terms.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package buda;
+
+enum PaymentTerms {
+  DUE_UPON_RECEIPT = 0;
+  NET_30 = 1;
+  NET_60 = 2;
+}
```

Comment:
I split `payment_terms` and `payment_schedule` into two different files, plus removed the message wrapper.  Does that maybe feel a bit cleaner, @michael-erasmus ?